demo/sizing.html

```diff
@@ -185,5 +185,106 @@
       </code>
     </code-example>
   </section>
+
+  <section>
+    <h3>Styling with Custom Properties</h3>
+    <p>The grid uses <code>--default-primary-color</code> from paper-styles as a highlight color.</p>
```

Comment:
Could have a link to the paper-styles GitHub repo, since some might not know what that is.